foodloop-db-files/01_schema.sql

```diff
@@ -1,6 +1,6 @@
-DROP DATABASE IF EXISTS foodloopdb;
-CREATE DATABASE foodloopdb;
-\c foodloopdb;
+-- DROP DATABASE IF EXISTS foodloopdb;
+-- CREATE DATABASE foodloopdb;
+-- \c foodloopdb;
```

Comment:
is this commented out?

Reply:
might possibly be caused by linter?

Reply:
have uncommented these lines and added `\c postgres` in Line 1 to switch connection to the `postgres` database before attempting to drop `foodloopdb`

this way, we would no longer be connected to `foodloopdb` while attempting to drop it